src/components/Skeleton/Skeleton.scss

```diff
@@ -0,0 +1,77 @@
+@import "../../styles/global-css-settings.scss";
+
+@mixin shine-animation() {
+  overflow: hidden;
+  &::before {
+    display: inline-block;
+    content:"";
+    opacity: 0.1;
+    background-image: linear-gradient(90deg, transparent 0%, var(--placeholder-color) 30%, var(--placeholder-color) 70%, transparent 100%);
+    width: 300%;
+    height: 300%;
+    margin-bottom: 20px;
+    animation-duration: 1.2s;
+    animation-fill-mode: forwards;
+    animation-iteration-count: infinite;
+    animation-name: shine;
+    animation-timing-function: $expand-animation-timing;
+  }
+  @keyframes shine {
+    0% {
+      margin-left: -500%;
+    }
+    100% {
+      margin-left: 500%;
+    }
+  }
+
+  // the better options which does not work
+  @keyframes shine2 {
+    0% {
+      background-position: 500px 0;
+    }
+
+    100% {
+      background-position: -500px 0;
+    }
+  }
+}
+
+@mixin skeleton($radius: $border-radius-small) {
+  @include theme-prop(background, disabled-background-color); // should be replaced with all grey color
+  border-radius: $radius;
+  @include shine-animation;
+
+}
+
+.monday-style-skeleton {
+  &_rectangle {
+    $rectangle-default-size: 160px;
+    height: $rectangle-default-size;
+    width: $rectangle-default-size;
+    @include skeleton;
+  }
+
+  &_circle {
+    $circle-default-size: 40px;
+    $circle-radius: 50%;
+    height: $circle-default-size;
+    width: $circle-default-size;
+    @include skeleton($circle-radius);
+  }
+
+  &_text {
+    width: 162px;
+    @include skeleton;
+
+    &--h1, &--h2, &--h3 {
```

Comment:
maybe you should use styles/typography.scss i.e. - .element-type-h1

Reply:
the size of the div elements are not identical to the given sizes. insted I add sass consts for the text skeleton heights in typography.scss